Add enterprise mode help content (#443)
cisagov/getgov-home: pages/domains/domains_enterprise-mode.md

sidenav: true
excerpt: Enterprise mode allows organizations to manage their domains and access
tags: domains
eleventyNavigation:
  key: domains
  order: 10
  title: Enterprise mode
---

Enterprise mode is designed for organizations that manage multiple .gov domains and want better visibility of their entire portfolio. It provides a holistic view of all domains and requests, and introduces roles and permissions to support detailed oversight.

- [Benefits of enterprise mode](#benefits-of-enterprise-mode)
- [Roles and permissions in enterprise mode](#roles-and-permissions-in-enterprise-mode)
- [What changes for existing users](#what-changes-for-existing-users)
- [Switching to enterprise mode](#switching-to-enterprise-mode)

## Benefits of enterprise mode

Enterprise mode provides a range of advantages, including:

### Centralized domain oversight for your organization.

View all your organization’s .gov domains in one dashboard, including which users manage each domain. You can also add your organization information (name, address, senior official) in one central location and apply it to all your organization’s domains.

### Selective control over permissions

> Grant only those permissions needed to basic members, like who in your organization can submit new domain requests to prevent unauthorized registrations.

Reviewer: ```suggestion
Decide exactly which actions each member can take, like who in your organization can submit new domain requests to prevent unauthorized registrations.
```

### Simplified domain request form

> Request a new domain more easily because your organization details are pre-filled, and our approval process is faster since only designated representatives can submit requests.

Reviewer: Looks great!

### Streamlined review process

Complete the review and approval process for new domains faster since your organization’s shared information only needs to be validated once.

## Roles and permissions in enterprise mode

> Enterprise mode introduces user roles and permissions to better manage access:

Reviewer: I think there's a way to streamline this content and combine with the next section for "additional permissions." I've made changes to this section in the [Google Doc](https://docs.google.com/document/d/1wgIydRN5KWwK8lWd1jWgh4SrYrGDAQhIsA6iawQ2Ejc/edit?tab=t.0#heading=h.tyjcwt) for your review. Note, I didn't track changes but I can replace with the original text if my ideas are rejected. cc: @h-m-f-t

### Organization admin

Full control over the organization’s domains, user roles, and permissions. This individual can see all registered domains for the organization and their settings, review domain managers assigned to each domain, and manage membership and permissions in the organization.

### Basic member

By default, can view and manage only domains they’ve been assigned to (also known as a “Domain Manager”). They can edit technical details, such as Name Servers and DNSSEC for domains they manage, add or remove domain managers from the domain, and edit the security email address for the domain.

#### Additional permissions

Basic members can be assigned additional permissions by an organization admin, including:

- View all organization domains and their details, not just the ones they manage  
- View organization domain requests  
- Make domain requests for the organization  
- View organization members

**How are admins and basic members different?**

<figure style="margin-left:0px !important;">

> <img alt="Org admins have all actions available to them. Basic members can only see the domain they manage unless an org admin gives additional permissions" src="../../uploads/images/enterprise-mode-admin-basic-comparison.png" title="Actions available to org admins and basic members" style="border: 1px solid;"/>

Reviewer: This table is an image; therefore, not accessible. We'll need dev to recreate this in code that can be accessed by screen readers.

Author: We could likely create a markdown/html table here instead of an image. I'll see how easily that can be done, or put in a table.

<figcaption style="font-style: italic; text-align:center; font-size:0.9em;">The org admin and basic member roles, with optional permissions</figcaption>
</figure>

## What changes for existing users

> * **Domain managers no longer have request privileges by default.** Only users explicitly granted “Requester” permissions can submit new domain requests.  

Reviewer: ```suggestion
* **Domain managers no longer have request privileges by default.** Only users explicitly granted “requestor” permissions can submit new domain requests.  
```

* **Organization information is shared across domains.** Updates to your organization’s profile (e.g., senior official, mailing address) automatically apply to all domains.  
* **Oversight of all domains is centralized.** Organization admins can view all domains and the users associated with each.  
* **The review process is simplified.** New domain requests may be approved more quickly since your organization’s details are already on file.

> ## Switching to enterprise mode

Reviewer: My earlier comment about the visual hierarchy between h2 and h3 applies here as well.

Author: Added the h4 styling on this section.

### Confirm your organization is eligible

> Enterprise mode is currently available for federal agencies. We expect to onboard state entities, large cities, and other organization types, in the future.

Reviewer: ```suggestion
Enterprise mode is currently available for federal agencies. We expect to onboard state entities, large cities, and other organization types in the future.
```
